packages/app/src/pages/EditorRoomPage.tsx

```
@@ -69,207 +112,188 @@ export const EditorRoomPage = () => {
+                  <div className="flex items-start justify-between">
+                    <div>
+                      <div className="font-heading text-lg">{d.title || "Untitled"}</div>
+                      <div className="text-xs text-text-secondary">Last saved: {formatDate(d.lastSaved || d.createdAt || Date.now())}</div>
+                    </div>
+                    <div className="flex items-center gap-2">
+                      <Button size="sm" variant="outline" onClick={() => openDraft(d.id)}>Edit</Button>
+                      <Button size="sm" variant="ghost" onClick={() => { if (confirm('Delete draft?')) removeDraft(d.id); }}>Delete</Button>
+                    </div>
+                  </div>
+                  <div className="mt-2 text-[15px] leading-relaxed text-text-primary">
+                    <div className="prose max-w-none overflow-hidden max-h-24" dangerouslySetInnerHTML={{ __html: renderMarkdownToHtml(d.content || "") }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)